### PR TITLE
add python 3.9, 3.10

### PR DESCRIPTION
**Pull request by Inveracity**

### CODE REVIEW

**Comment by HenrikPoulsen** [comment]
Would be nice if someone could take a look at this.
I have repos that cannot run this action now because we need to use python 3.10

**Comment by Inveracity** [comment]
@garethr sorry to ping you, but you seem to be the one updating this repo, mind taking a look? Thanks 🙂

File: build.rb [hunk 1]
```diff
@@ -27,6 +27,9 @@
   "Python-3.6",
   "Python-3.7",
   "Python-3.8",
+  "Python-3.9",
+  "Python-3.10",
+  "Python-3.11",
```
Comment (bastiandoetsch) [inline build.rb:32]: There is no snyk image available for Python 3.11. The action thus wouldn't work. Would you please remove 3.11?

Comment (Inveracity) [reply build.rb]: done 👍

**Comment by bastiandoetsch** [comment]
Thanks for the contribution!

**Pull request merged**